scrypto/src/component/component.rs

```diff
@@ -138,17 +141,6 @@ impl BorrowedGlobalComponent {
         let state: DataRef<ComponentInfoSubstate> = pointer.get();
         state.blueprint_name.clone()
     }
-
-    pub fn add_access_check(&mut self, access_rules: AccessRules) -> &mut Self {
```

Comment:
No more `add_access_check` for globalized components. I like it!

Reply:
I think this is temporary and will need to add back in the future :)